swap_meet/vendor.py

```diff
@@ -1,2 +1,90 @@
+# from swap_meet.item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
Let's flip the order of the conditional check. That way we can bring the main focus of the method out of the indentation.
```python
    if item not in self.inventory:
        return False

    self.inventory.remove(item)
    return item
```